lib/stdlib/test/sys_SUITE.erl

```diff
@@ -146,6 +146,20 @@ install(Config) when is_list(Config) ->
     {ok,-4} = (catch public_call(4)),
     [{spy_got,{request,1},sys_SUITE_server},
      {spy_got,{request,3},sys_SUITE_server}] = get_messages(),
+
+    sys:install(?server,{id1, SpyFun, func_state}),
+    sys:install(?server,{id1, SpyFun, func_state}), %% should not be installed
+    sys:install(?server,{id2, SpyFun, func_state}),    
+    {ok,-1} = (catch public_call(1)),
+    %% We have two SpyFun installed:
+    [{spy_got,{request,1},sys_SUITE_server},
+     {spy_got,{request,1},sys_SUITE_server}] = get_messages(),
+    sys:remove(?server, id1),
+    {ok,-1} = (catch public_call(1)),
+    %% We have one SpyFun installed:
+    [{spy_got,{request,1},sys_SUITE_server}] = get_messages(),
+    sys:no_debug(?server),
```

Comment:
Maybe call public_call/1 after this also? Then next get_messages() should still return the empty list then...

Reply:
What line should has another `public_call`? I added it after every installing/removing.

Reply:
I was thinking between `sys:no_debug(?server)` and `[]=get_messages()`, to check that it is turned off again. I'ts a minor though - just thought that the call to `get_messages()` on line 162 would be kind of meaningless otherwise. Or did I miss something more??

Reply:
Cool. got it.